sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -75,6 +75,25 @@ var credential = new AzureKeyCredential(apiKey);
 var client = new FormRecognizerClient(new Uri(endpoint), credential);
 ```
 
+#### Create FormRecognizerClient with Azure Active Directory Credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain] for your resource in order to use this type of authentication.
```

Comment:
Does FR support regional endpoints?  In the Azure Portal, I only see the option to create a resource with a custom subdomain.

Reply:
Double checked with Chris:
```
yes, we support regional endpoints. your named endpoint is actually a cname for the regional host
```